.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - master
+
+env:
+  BAHMNI_VERSION: 0.94
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Install dependencies
+        run: yarn install
+      - name: Run tests
+        run: yarn test
+      - name: Package
+        run: yarn ci
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: implementer-interface-${{ github.sha }}
+          path: ./dist/implementer_interface.zip
```

Comment:
why do we need to upload the dist here since the docker build and publish is in the same job?